code/game/objects/items/devices/techno_tribalism.dm

```diff
@@ -2,6 +2,8 @@
 	name = "Techno-Tribalism Enforcer"
 	desc = "Feeds on rare tools, tool mods and other tech items. After being fed enough, will produce a strange technological part, that will be useful to train your skills overtime."
 	icon = 'icons/obj/faction_item.dmi'
+	description_info = "Has a internal radio that informs technomancers of its delay, it can be reenabled with a screwdriver if its not functioning. Deconstructing other departamental oddities reduces its cooldown"
```

Comment:
```suggestion
	description_info = "Has an internal radio that informs technomancers of its delay, it can be reenabled with a screwdriver if it is not functioning. Deconstructing other departamental oddities reduces its cooldown."
```